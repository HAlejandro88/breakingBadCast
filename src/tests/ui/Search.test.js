import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import Search from '../../components/ui/Search';

describe('pruebas en el componente <Search/>', () => {
    const getQuery = jest.fn();
    let wrapper = shallow(<Search getQuery={getQuery}></Search>)
    beforeEach(() => {
        jest.clearAllMocks();
        wrapper = shallow(<Search getQuery={getQuery}></Search>)
    });
    test('pruebas contra el snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });

});