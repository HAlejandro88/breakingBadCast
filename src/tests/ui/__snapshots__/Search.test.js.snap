// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en el componente <Search/> pruebas contra el snapshot 1`] = `
<section
  className="search"
>
  <form>
    <input
      autoFocus={true}
      className="form-control"
      onChange={[Function]}
      placeholder="Search character"
      type="text"
      value=""
    />
  </form>
</section>
`;
